truffle/com.oracle.truffle.api.debug/src/com/oracle/truffle/api/debug/ExecutionEvent.java

```diff
@@ -87,9 +90,11 @@ public Debugger getDebugger() {
      * </ul>
      * 
      * @since 0.9
+     * @deprecated Continue is by default, this method is not necessary.
      */
```

Comment:
Well, you have just a single event handler callback to "change your mind", thus I considered this method unnecessary.
But O.K. if you'd like to keep it, I'll change the code accordingly.